data/campaigns/Sceptre_of_Fire/scenarios/6_Towards_the_Caves.cfg

```diff
@@ -620,7 +620,7 @@
                 [/message]
                 [message]
                     speaker=Rugnur
-                    message= _ "Tell him to send forces north to aid us and fight the elves, but that we are probably going to die. We will run northeast towards the old eastern mines, and if he wants his sceptre intact he’d best send troops to meet us there."
+                    message= _ "Tell him to send forces north to aid us and fight the elves, but that we are probably going to die. We will run northeast towards the old eastern mines. And if he wants his sceptre intact, he’d best send troops to meet us there."
```

Comment:
Change "And if" to "If"?

I know the PR is already merged, but it's a handy place to add more comments for whatever will happen next. This isn't the only sentence in this campaign to start with "And", other existing ones might need a review too.

Reply:
If the original author put the word "and" here, I say keep it. It's not too hard to believe that in oral speech someone would put and in that place while he is talking.
If this text represented only written text, sure it makes sense to really strike out the "and".